wasswa-derick/NairobiJavaGeeks
# [Feature #157354404] Add Instrumentation tests (UI tests)

#### What does this PR do?
* UI Test the Code.

#### Description of Task to be completed?
*  Implement UI tests using Espresso (instrumentation tests) with a real device or an emulator.
#### How should this be manually tested?
* Clone the project, ```git clone https://github.com/wasswa-derick/NairobiJavaGeeks.git```.
* In your terminal, ```cd NairobiJavaGeeks```
* Checkout to the ```ft-ui-tests-157354404``` branch.
* Build the Project and ensure the Gradle build completes successfully.
* From the Gradle Panel run the ```jacocoTestReport``` task which shall run the tests and give you a report in the ```app/build/jacoco``` directory.


#### What are the relevant pivotal tracker stories?
[#157354404](https://www.pivotaltracker.com/story/show/157354404)

#### ScreenShots

> Jacoco Test Report

<img width="1249" alt="jacocoreport" src="https://user-images.githubusercontent.com/39955231/46353342-98f4a800-c664-11e8-9ffd-4d50db682871.png">
